.github/workflows/version-bump.yml

```diff
@@ -0,0 +1,16 @@
+name: Bump version
+on:
+  push:
+    branches:
+      - main
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Bump version and push tag
+        uses: hennejg/github-tag-action@v4.1.jh1
```

Comment:
I think we should use this one instead: https://github.com/marketplace/actions/github-tag

Not sure if how much of a difference there is between the two, but this has more stars and seems to be updated more frequently. Also the one I linked originally (the one you used) was forked from this marketplace action.

There's also instructions on that page to automate releases should we want to do that in the future.

Reply:
i think those are good reasons to use this one instead!

Reply:
Yeah damn, the docs are basically the exact same too, just slight variations in configuration 😂

Reply:
Updated!